fix(docs): content review (#2050)
scaleway/docs-content: tutorials/migrate-databases-instance/index.mdx

### Preparing the Database for PostgreSQL

<Message type="note">

>   In case you do not have a Database for PostgreSQL yet, [create a new one](/managed-databases/postgresql-and-mysql/how-to/create-a-database/) from your [Scaleway console](https://console.scaleway.com/rdb/instances).

Reviewer: ```suggestion
  If you do not have a Database for PostgreSQL yet, [create a new one](/managed-databases/postgresql-and-mysql/how-to/create-a-database/) from your [Scaleway console](https://console.scaleway.com/rdb/instances).
```

### Preparing the Database for MySQL

<Message type="note">

>   In case you do not have a Database for MySQL yet, [create a new one](/managed-databases/postgresql-and-mysql/how-to/create-a-database/) from your [Scaleway console](https://console.scaleway.com/rdb/instances).

Reviewer: ```suggestion
  If you do not have a Database for MySQL yet, [create a new one](/managed-databases/postgresql-and-mysql/how-to/create-a-database/) from your [Scaleway console](https://console.scaleway.com/rdb/instances).
```
